content/en/docs/security/otel-collector-hosting-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: OpenTelemetry Collector Hosting Best Practices
+description: Follow best practices to securely host OpenTelemetry Collector.
+weight: 10
+---
+
+When setting up hosting for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your hosting instance.
+
+## Storing Configuration Information Securely
+
+<!--- TODO: SHOULD ensure sensitive configuration information is stored securely. How? -->
+
+## Permissions
+
+<!--- TODO: SHOULD not run the OpenTelemetry Collector as root/admin user. Why? (Give the reader motivation.) How do you do that?
+- NOTE: MAY require privileged access for some components -->
+
+## Receivers and Exporters
+
+<!--- TODO: SHOULD limit exposure of servers to authorized users. How do you do that? -->
```

Comment:
- Authentication (bearer token auth extension, basic auth extension)
- By restricting the IPs the collector is running